tests/test_middlewares.py

```diff
@@ -1,15 +1,15 @@
 # -*- coding: utf-8 -*-
+import itertools
+import sys
 from weakref import WeakKeyDictionary
 
-import itertools
 import pytest
-from scrapy import Spider, Request, Item
+from scrapy import Item, Request, Spider
 from scrapy.http import Response
 
-from sh_scrapy.middlewares import (
-    HubstorageSpiderMiddleware, HubstorageDownloaderMiddleware,
-    HS_REQUEST_ID_KEY, HS_PARENT_ID_KEY
-)
+from sh_scrapy.middlewares import (HS_PARENT_ID_KEY, HS_REQUEST_ID_KEY,
+                                   HubstorageDownloaderMiddleware,
+                                   HubstorageSpiderMiddleware)
```

Comment:
Minor:

I'm not so sure about this reformatting.

We usually use isort on our repos which has a different structure than this. It would seem this repo isn't configure to have such linter. 

I think it'd be better to revert this change. The commit message also is different https://github.com/scrapinghub/scrapinghub-entrypoint-scrapy/pull/73/commits/e7a586534d6eb56ea35480cf7c90d8e128e03189.

What do you think?

Reply:
Yes, agree, I've just applied my local isort automatically - will change this back.